lib/ffmpeg/qsv/decoder.py

```diff
@@ -15,6 +15,10 @@
 class Decoder(FFDecoder):
   hwaccel = property(lambda s: "qsv")
 
+  @property
+  def hwinit_format(self):
+    return f"qsv=qsv,child_device={self.hwdevice}"
```

Comment:
Could we just override `hwdevice`?

```python
@property
def hwdevice(self):
  return f"hw_any,child_device={super().hwdevice}"
```
... or similar

Reply:
hmm... what is `hw` vs. `qsv` in this part?  Is it just a arbitrary name or does it actually map to something special?

Reply:
It's annoying that there are so many different syntax between qsv, vaapi and other's.  @xhaihao is there a common/normalized syntax that is supported universally by all hwaccel plugins?

Reply:
@xhaihao The different syntax options are confusing to end-users.  Please make it simple.

Reply:
the format is `type=name:device,key1=xxx,...`, you may use any valid string for name, e.g. hw, vaapi, qsv, intel, etc.

Reply:
"any valid string"?  So this implies the "name" maps to something specific?  Or can it be arbitrary, like "abc" or "123"?  And how is the "name" used?

Reply:
> "any valid string"? So this implies the "name" maps to something specific? Or can it be arbitrary, like "abc" or "123"? And how is the "name" used?

It can be arbitrary, you may name the device with "abc" or "123" if you like these names. You may specify the device name for decoding via option `-hwaccel_device` and for filters via option`-filter_hw_device`. 

e.g. device 'abc' and '123' are created in the below command, and '123' is chosen for decoding & filters. 

$ ffmpeg -init_hw_device qsv=abc -init_hw_device qsv=123 -hwaccel qsv -hwaccel_device 123 -filter_hw_device 123  -i input.mp4 -f null -

You may derive a new device from a named child device. 

e.g. create a vaapi device with name 'va', then derive a qsv device with name 'abc', device 'abc' is used for decoding and filters. 
$ ffmpeg -init_hw_device vaapi=va -init_hw_device qsv=abc@va -hwaccel qsv -hwaccel_device abc -filter_hw_device abc  -i input.mp4 -f null -